### DISCUSSION TITLE
v0.15.4

### DISCUSSION THREAD

**Discussion by alexjustesen** [alexjustesen/speedtest-tracker#1133] [Releases]
## What's Changed
* [Bug] Fixed testing Telegram notification channel by @alexjustesen in https://github.com/alexjustesen/speedtest-tracker/pull/1130
* Laravel 10.44.0 Shift by @alexjustesen in https://github.com/alexjustesen/speedtest-tracker/pull/1128
* Release v0.15.4 by @alexjustesen in https://github.com/alexjustesen/speedtest-tracker/pull/1132

**Full Changelog**: https://github.com/alexjustesen/speedtest-tracker/compare/v0.15.3...v0.15.4

<hr /><em>This discussion was created from the release <a href='https://github.com/alexjustesen/speedtest-tracker/releases/tag/v0.15.4'>v0.15.4</a>.</em>

**Comment by CrsMthw** [comment]
Thank you for all your hardwork in making this beautiful piece of software. 🙏 Easily one of my most favorite no-nonsense project in recent years. Thanks again! 🫡

**Reply by alexjustesen** [reply]
Thank you!
